Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -1424,6 +1453,13 @@ std::vector<std::vector<unsigned int>> getMolSelfMatches(
 
 void EmbedMultipleConfs(ROMol &mol, INT_VECT &res, unsigned int numConfs,
                         EmbedParameters &params) {
+  TimePoint* end_time = nullptr;
+  std::optional<TimePoint> end_time_opt;
```

Comment:
I'm curious why this is a `std::optional` and not just a `TimePoint`.

Reply:
Right, can't think of a clear benefit; changed it to just a `TimePoint`.